### PR TITLE
Reports controller 

### PR DESCRIPTION
**Pull request by tarunguguloth**
added reports controller and request response body

### CODE REVIEW

File: internal/controller/reports.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import (
+	"Tradeasy/internal/model"
+	"Tradeasy/internal/services/reports"
+	"fmt"
+	"github.com/gin-gonic/gin"
+	"net/http"
+)
+
+func DailyPendingOrders(c *gin.Context){
+	var rep []model.PendingOrders
```
Comment (mukesh0513) [inline internal/controller/reports.go]: lets use requests structs same as other models.

File: internal/controller/reports.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+	id := c.Params.ByName("id")
+	c.BindJSON(&rep)
```
Comment (mukesh0513) [inline internal/controller/reports.go]: handle bind error.

File: internal/services/reports/reports.go [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+}
+func OrdersHistory(report1 []model.OrderHistory,report2 []model.Holdings, Userid string,from string,to string) (ordHisRes OrderHistoryResponse,err error){
+
+	return ordHisRes,nil
+}
+func ProfitLossHistory(reports []model.OrderHistory, Userid string, from string,to string) (proLosRes ProfitLossHistoryResponse,err error){
+
+	return proLosRes,nil
+}
+
```
Comment (mukesh0513) [inline internal/services/reports/reports.go:15]: remove extra lines.

File: internal/controller/reports.go (hunk 1 above)
Comment (tarunguguloth) [reply internal/controller/reports.go]: This is a get method and I am directly getting all the orders from pending orders table right ..So, why to use the response struct again and make the same struct as pending orders?

**Comment by tarunguguloth** [comment]
merged reports_controller PR to payments_controller PR

**Pull request closed without merging**